### PR TITLE
Initial repo setup

### PR DESCRIPTION
**Pull request by rnathuji**

### CODE REVIEW

**Review by mjoss17 — APPROVED** [review]

**Comment by rnathuji** [comment]
Adding a note to reflect conversation during pairing session: This initial PR is going to fail CI as `pytest` returns a non-zero error code due to the lack of tests. We'll merge this in despite that (once approved) and will expect CI to pass on a subsequent PR where the tests are added.

**Pull request merged**
